backend/routes/user.js

```diff
@@ -12,6 +19,36 @@ import {
 const routes = express.Router();
 const upload = multer({ storage: multer.memoryStorage() }).array('images');
 
+routes.put(
+  '/update-driver-verification-status',
```

Comment:
So we don't have to make a API request for each individual user, can we pass in the userIds as a list of users? Additionally, it is probably better to name the route something like /driverVerificationStatus since the PUT label on the route already suggests that the operation will be an update.